packages/venia-concept/src/components/SignIn/signIn.js

```diff
@@ -76,22 +86,15 @@ class SignIn extends Component {
     }
 
     onSignIn = () => {
-        const { username, password } = this.state;
-        this.props.signIn({ username: username, password: password });
+        const username = this.getFieldValue(fields.username);
```

Comment:
If you pass onSignIn function to onSubmit will receive `values` object as an argument.